test/livebook_web/live/hub/edit_live_test.exs

```diff
@@ -63,6 +63,38 @@ defmodule LivebookWeb.Hub.EditLiveTest do
       refute Hubs.fetch_hub!(hub.id) == hub
     end
 
+    test "deletes hub", %{conn: conn, bypass: bypass} do
+      {:ok, pid} = Agent.start(fn -> %{fun: &fetch_app_response/2, type: :mount} end)
+      app_id = Livebook.Utils.random_short_id()
+      hub_id = "fly-#{app_id}"
+
+      hub = insert_hub(:fly, id: hub_id, application_id: app_id)
+      fly_bypass(bypass, app_id, pid)
+
+      {:ok, view, html} = live(conn, Routes.hub_path(conn, :edit, hub.id))
+
+      assert html =~ "Manage app on Fly"
+      assert html =~ "https://fly.io/apps/#{hub.application_id}"
+
+      assert html =~ "Environment Variables"
+      refute html =~ "FOO_ENV_VAR"
+      assert html =~ "LIVEBOOK_PASSWORD"
+      assert html =~ "LIVEBOOK_SECRET_KEY_BASE"
```

Comment:
I'd keep the test more focused, since those assertions are not related to deletion :)

```suggestion
```